cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -302,7 +302,6 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
         val licenseInfoResolver = LicenseInfoResolver(
             provider = DefaultLicenseInfoProvider(ortResultInput, packageConfigurationProvider),
             copyrightGarbage = copyrightGarbage,
-            addAuthorsToCopyrights = config.addAuthorsToCopyrights,
```

Comment:
It was decided during PR grooming by the TSC that an implementation of the general feature to conclude copyrights would at first *not* remove the feature to map authors to copyrights. This is to allow users for a transition period to migrate configuration from one feature to the other smoothly.